analytical_engine/java/README.md

```diff
@@ -201,15 +201,15 @@ graph = graph.add_vertices("gstest/property/p2p-31_property_v_0", label="person"
 graph = graph.add_edges("gstest/property/p2p-31_property_e_0", label="knows")
 
```

Comment:
This README needs to revise. (maybe in another PR)
- maybe guide users to use built-in app;
- and then implement their own app, then build and run.

Reply:
Ok, I will revise the README in another PR.